langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibArrayTest.java

```diff
@@ -247,6 +247,23 @@ public void testIndexOf() {
         assertNull(returns[1]);
     }
 
+    @Test
+    public void testLastIndexOf() {
+        BValue[] returns = BRunUtil.invoke(compileResult, "testLastIndexOf");
```

Comment:
Shall we write the tests in Ballerina itself?

Reply:
Added